### DISCUSSION TITLE
Next Design compatibility

### DISCUSSION THREAD

**Discussion by dawiidio** [lumada-design/hv-uikit-react#4472] [Q&A] [answered]
Hello, I'm currently  working on another Hitachi project where we use NextUI and have plans to use it in our future projects, but recently a news reached me, that NextUI may no longer support Next Design System. Would you be able to share information about this? Thanks!

**Comment by francisco-guilherme** [comment, marked as answer]
Hi @dawiidio  👋

Thank you for opening this discussion topic! While we’ve been addressing this through other channels, it’s essential to have a public discussion here, as it’s an important topic for the community.

The UI Kit team has recently been focusing on adopting the new Pentaho Design System!
This doesn’t mean that the Next Design System will no longer be supported, but this shift has introduced some challenges. As a result, we’ve temporarily paused updates to the Next Design System.

One of our primary goals moving forward is to enhance our theming capabilities, enabling streamlined support for multiple design systems. This opens up exciting possibilities for other Hitachi organizations to integrate and benefit from our work.

Being an open-source project with a proven contribution model, we’re more than willing to collaborate with the community to support development efforts, address missing features, and drive improvements.

If you’re interested, please reach out to us! Let’s set up a time to chat—we’d be happy to walk through the current state of the UI Kit and explore ways to collaborate to overcome any existing challenges.

**Reply by dawiidio** [reply]
Hello Francisco, Thank you very much for you answer! 

I am pleased to hear that you are not withdrawing support for Next DS and are open to cooperation. Also, I think it would be a great initiative to create a open-source implementation of design system and its further adoption in the organization. 
This will be on my agenda in the coming days while planning the work for the coming year. I will get back to you on setting up a meeting to discuss the details.
